packages/block-library/src/search/block.json

```diff
@@ -83,6 +91,7 @@
 		},
 		"html": false
 	},
+	"viewScript": "file:./view.js",
```

Comment:
```suggestion
	"viewScript": "file:./view.min.js",
```

Should this be the `.min` version of the script? I see this is the version used in the [file block](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/file/block.json#L70) and the [navigation block](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/navigation/block.json#L137).

I'm also seeing a PHP notice on this branch, which the `.min` version of this script seems to fix:

```
Notice: Function register_block_script_handle was called <strong>incorrectly</strong>. The asset file (/build/block-library/blocks/search/view.asset.php) for the "viewScript" defined in "core/search" block definition is missing.
```